goss-testing/tests/ncn/goss-bond0-ip-validation.yaml

```diff
@@ -0,0 +1,41 @@
+#
```

Comment:
The test `goss-interface-has-ip.yaml` already verifies that several interfaces on the system have IPv4 IP addresses, including bond0.cmn0. Is there a need for a new test to test specifically just this interface?